CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# Release 0.26.12
+
+## What's New
+
+- Ziti CLI
+  - Added CLI flags for setting router tunneler capability
+
+## Ziti CLI
+### Added CLI flags for setting router tunneler capability
+Ziti CLI `ziti create config router edge` now has three new flags
+#### --disableTunneler
+The tunneler capability for routers is enabled by default, the `disableTunneler` flag can now be used to disable the 
+tunneler capability in the router config.
```

Comment:
what happens when i supply --disableTunneler and then set the mode/lanIf? that should be called out here imo

Reply:
actaully - feels like maybe --disableTunneler should just be removed and three 'modes' supported: none, host, tproxy